styles/styles.css

```diff
@@ -1332,4 +1332,17 @@ body.no-scroll {
 .auto-image-container {
   padding:0;
   margin:0;
-}
\ No newline at end of file
+}
+
+#who-we-are{
+  margin-bottom: var(--spacing-small);
+  font-size: var(--font-size-1200);
+}
+
+#who-we-are + p  {
+  margin-top: 0;
+}
+
+#who-we-are+p+p {
+  margin-bottom: 0;
+}
```

Comment:
This `id` changes based on the authored content. This might also not get reflected in translated languages? I think @skumawat1 worked on some fix here as part of bugfix in the ticked. Could you please connect with him to know which issues are handled?

Reply:
Done. Removed the styles from here and added to heading.css